src/components/ExpandCollapse/ExpandCollapse.module.scss

```diff
@@ -0,0 +1,76 @@
+@import "../../styles/themes";
+@import "../../styles/typography";
+@import "../../styles/states";
+@import "../../styles/global-css-settings";
+
+.wrapper .expandCollapse {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  @include theme-prop(color, primary-text-color);
+  background-color: transparent;
+  border: 1px solid;
+  @include theme-prop(border-color, ui-border-color);
+  box-sizing: border-box;
+  border-radius: $border-radius-small;
+}
+
+.wrapper .header {
```

Comment:
do we need nesting? if so , why not use scss nesting?

Reply:
Orr's request was to remove scss nesting